site/public/css/electronic.css

```diff
@@ -0,0 +1,1236 @@
+    text-align: right;
+    width: 100%;
+    margin: 2px;
+}
+
+.gradeable .mark-container input:disabled {
+    background-color: transparent;
+    border: none;
+}
+
+.gradeable .mark-container.mark-publish {
+    background-color: #bde;
+}
+
+.gradeable .mark-disabled span {
+    cursor: not-allowed;
+    opacity: 0.3;
+}
+
+.gradeable .mark-deleted {
+    background: #FFAAAA
+}
+
+.gradeable .custom-mark-error {
+    border-color: #F00;
+    background-color: #FEE;
+}
+
+.gradeable .mark-deleted span.delete-mark-container {
+    display: none;
+}
+
+.gradeable span.restore-mark-container {
+    display: none;
+}
+
+.gradeable .mark-deleted span.restore-mark-container {
+    display: inherit;
+}
+
+.gradeable span.reorder-mark-container {
+    min-width: 20px;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.gradeable span.mark-tools {
+    min-width: 20px;
+    width: 20px;
+    text-align: right;
+    opacity: 1.0;
+}
+
+.gradeable .mark-uneditable {
+    cursor: default;
+    border: none;
+    outline: none;
+    background-color: initial;
+}
+
+.mark-container .col-sm input {
+    width: 100%;
+    margin: 2px;
+}
+
+.gradeable .add-new-mark {
+    margin-left: 25px;
+}
+
+.gradeable .received-marks-list.container {
+    padding-left: 15px;
+    padding-top: 0;
+}
+
+.gradeable .received-marks-list.container .row .col-no-gutters.point-value,
+.gradeable .received-marks-list.container .row .col-no-gutters.indicator {
+    display: inline-flex;
+    align-items: center;
+    padding-left: 5px;
+    padding-right: 5px;
+    width: 30px;
+}
+
+.gradeable .received-marks-list.container .row .col-no-gutters.point-value {
+    justify-content: flex-end;
+}
+
+.gradeable .received-marks-list.container .row .col-no-gutters.indicator {
+    justify-content: center;
+}
+
+/* Mark conflict popup */
+
+#mark-conflict-popup div.mark-conflict-row {
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+
+#mark-conflict-popup .mark-conflict-row span {
+    padding: 3px;
+    border-width: 2px;
+    margin: 0;
+    display: inline-flex;
+    align-items: center;
+}
+
+#mark-conflict-popup .mark-conflict-row .button-container {
+    width: 150px;
+}
+
+#mark-conflict-popup .mark-conflict-row .button-container input {
+    width: 100%;
+}
+
+#mark-conflict-popup .mark-conflict-row.mark-resolved,
+#mark-conflict-popup .mark-conflict-row.hidden {
+    display: none;
+}
+
+#mark-conflict-popup .conflict-resolve-progress {
+    width: fit-content;
+    margin: 0 auto;
+}
+
+#mark-conflict-popup .mark-conflict-container {
+    margin-top: 25px;
+}
+
+#discussion_browser.rubric_panel {
+    border:5px solid var(--standard-light-seafoam);
+}
+
+#directory_view {
+    position:relative;
+    float:left;
+    width:100%;
+}
+
+#changeCodeStyle {
+    display:inline-block;
+}
+
+/* Animations */
+@keyframes scaleUp {
+    from {
+        position: fixed;
+        height: 40vh;
+        top: 30%;
+        left: 20%;
+    }
```

Comment:
I don't know much about css animation, but can we make the panels start from their current position rather than specifying a starting position?

Reply:
added a little change in animation